training_record/tests/tests_views.py

```diff
@@ -116,7 +116,7 @@ def test_get_record(self):
         url = reverse('record-detail', args=(record.pk,))
         expected_keys = {'id', 'create_time', 'update_time', 'campus_event',
                          'off_campus_event', 'user', 'status', 'contents',
-                         'attachments', 'status_str', 'feedback'}
+                         'attachments', 'status_str'}
```

Comment:
You should keep the `feedback` field on the serializer, otherwise you can't tell whether there is a `feedback` related to the `record`.